_docs/index.html

```diff
@@ -111,6 +111,10 @@ <h2>Distributions</h2>
         <img src="{{ '/images/wiremock-cloud/wiremock_cloud_favicon.svg' | absolute_url }}">
         WireMock Cloud (commercial SaaS)
     </a>
+    <a class="card" href="./solutions/nodejs">
```

Comment:
I suggest pointing to https://www.npmjs.com/package/wiremock instead

```suggestion
    <a class="card" href="https://www.npmjs.com/package/wiremock"  target="_blank">
```